integrations/Vero/browser.js

```diff
@@ -0,0 +1,155 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable class-methods-use-this */
+import get from "get-value";
+import { NAME } from "./constants";
+import logger from "../../utils/logUtil";
+import { isDefinedAndNotNull } from "../utils/commonUtils";
+import ScriptLoader from "../ScriptLoader";
+
+class Vero {
+  constructor(config) {
+    this.apiKey = config.apiKey;
+    this.name = NAME;
+  }
+
+  init() {
+    logger.debug("===In init Vero===");
+    window._veroq = window._veroq || [];
+    ScriptLoader(null, "https://d3qxef4rp70elm.cloudfront.net/m.js", true);
```

Comment:
Add some ID for the script.

Reply:
`async` defaults to `true`. So, no need to send the argument if it is `true`.

Reply:
Done

Reply:
done